keps/sig-instrumentation/383-new-event-api-ga-graduation/README.md

```diff
@@ -439,6 +449,198 @@ List Events from the NamespaceSystem with field selector `reportingController =
+  logs or events for this purpose.
+
+  There's no way to tell that through an operator. If a component has been migrated to use the new Event API and the API is enabled, it should use that by default. Currently only scheduler has been migrated.
+
+* **What are the SLIs (Service Level Indicators) an operator can use to
+  determine the health of the service?**
+  - [ ] Metrics
+    - Metric name:
+    - [Optional] Aggregation method:
+    - Components exposing the metric:
+  - [ ] Other (treat as last resort)
+    - Details:
+
+* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+  At the high-level this usually will be in the form of "high percentile of SLI
+  per day <= X". It's impossible to provide a comprehensive guidance, but at the very
+  high level (they needs more precise definitions) those may be things like:
+  - per-day percentage of API calls finishing with 5XX errors <= 1%
+  - 99% percentile over day of absolute value from (job creation time minus expected
+    job creation time) for cron job <= 10%
+  - 99,9% of /health requests per day finish with 200 code
+
+* **Are there any missing metrics that would be useful to have to improve
+  observability if this feature?**
+  Describe the metrics themselves and the reason they weren't added (e.g. cost,
+  implementation difficulties, etc.).
+
+  No.
+
+### Dependencies
+
+_This section must be completed when targeting beta graduation to a release._
+
+* **Does this feature depend on any specific services running in the cluster?**
+  Think about both cluster-level services (e.g. metrics-server) as well
+  as node-level agents (e.g. specific version of CRI). Focus on external or
+  optional services that are needed. For example, if this feature depends on
+  a cloud provider API, or upon an external software-defined storage or network
+  control plane.
+
+  For each of the fill in the following, thinking both about running user workloads
+  and creating new ones, as well as about cluster-level services (e.g. DNS):
+  - Etcd
+    - Usage description:
+      - Impact of its outage on the feature: New Events generation mechanism might overwhelm etcd.
+      - Impact of its degraded performance or high error rates on the feature: Edcd might not be able to handle increasing amount and size of Events.
+
+
+### Scalability
+
+_For alpha, this section is encouraged: reviewers should consider these questions
+and attempt to answer them._
+
+_For beta, this section is required: reviewers must answer these questions._
+
+_For GA, this section is required: approvers should be able to confirms the
+previous answers based on experience in the field._
+
+* **Will enabling / using this feature result in any new API calls?**
+  Describe them, providing:
+
+  In the new EventRecorder, every 30 minutes a "heartbeat" call will be performed to update Event status and prevent garbage collection in etcd. This is only for isomorphic Events, which won't happen too frequently.
```

Comment:
We need to be more precise here.
This heartbeat is happening for events that are generated "continuously" (if an event didn't happen for 6 minutes? @yastij - do you remember if it is 6?), it will be GC-ed anyway.
So it's only for events that are happening all the time (and we simply don't record every single instance of them).

Reply:
@wojtek-t - we do close series of event after 6min. and we refresh TTL on keys each 30min

Reply:
Improved the docs.